.github/workflows/release.yml

```diff
@@ -10,79 +10,217 @@ jobs:
+
+      - if: ${{ matrix.os == 'macos-12' &&  matrix.arch == 'native'}}
+        name: Setup gfortran for Macos
+        run: |
+          set -xe
+          # Taken from https://github.com/numpy/numpy/tree/main/tools/wheels
+          chmod +x .github/workflows/gfortran_utils.sh
+          source .github/workflows/gfortran_utils.sh
+          install_gfortran
+          gfortran --version
+          which gfortran
+
+
       - if: ${{ matrix.arch == 'aarch64' }}
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v2
+
+      - if: ${{ matrix.os != 'macos-12' }}
+        uses: awvwgk/setup-fortran@main
+        id: setup-fortran
+        with:
+          compiler: gcc
+          version: 11
+
+      - if: ${{ matrix.os == 'windows-latest' }}
+        uses: pypa/cibuildwheel@v2.11.2
+        env:
+          CIBW_BUILD_VERBOSITY: 3
+          CIBW_BUILD: ${{ matrix.py }}-*
+          CIBW_ARCHS: ${{ matrix.arch }}
+          CIBW_SKIP: "*musllinux*"
+          CMAKE_GENERATOR: "MinGW Makefiles"
+          FC: ${{ steps.setup-fortran.outputs.fc }}
+
+      - if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: pypa/cibuildwheel@v2.11.2
+        env:
+          # Check setuptools version
+          CIBW_BUILD_VERBOSITY: 3
+          CIBW_BUILD: ${{ matrix.py }}-*
+          CIBW_ARCHS: ${{ matrix.arch }}
+          CIBW_SKIP: "*musllinux*"
 
-      - uses: pypa/cibuildwheel@v2.3.1
+      - if: ${{ matrix.os == 'macos-12' &&  matrix.arch == 'native' }}
+        uses: pypa/cibuildwheel@v2.11.2
         env:
+          # Check setuptools version
+          CIBW_BUILD_VERBOSITY: 3
           CIBW_BUILD: ${{ matrix.py }}-*
           CIBW_ARCHS: ${{ matrix.arch }}
+          CIBW_SKIP: "*musllinux*"
+          no_proxy: '*'
+
+      - if: ${{ matrix.os == 'macos-12' &&  matrix.arch == 'arm64' }}
+        id: cibw_step
+        uses: pypa/cibuildwheel@v2.11.2
+        env:
+          # Check setuptools version
+          CIBW_BUILD_VERBOSITY: 3
+          CIBW_BUILD: ${{ matrix.py }}-*
+          CIBW_ARCHS: ${{ matrix.arch }}
+          CIBW_SKIP: "*musllinux*"
+          CIBW_TEST_SKIP: "*_arm64 *_universal2:arm64"
+          CIBW_ARCHS_MACOS: arm64
+          CIBW_BEFORE_BUILD: |
+            set -xe
+            # Taken from https://github.com/numpy/numpy/tree/main/tools/wheels
+            export PLAT="arm64"
+            chmod +x .github/workflows/gfortran_utils.sh
+            source .github/workflows/gfortran_utils.sh
+            install_arm64_cross_gfortran
+          FC: "/opt/gfortran-darwin-arm64-cross/bin/arm64-apple-darwin20.0.0-gfortran"
+          MACOSX_DEPLOYMENT_TARGET: "11.0"
+          no_proxy: '*'
 
       - uses: actions/upload-artifact@v3
         with:
           path: ./wheelhouse/*.whl
 
-  sdist:
-    name: source package
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: true
+  # sdist:
```

Comment:
The sdist is needed, I hope it will come back and was only temporarily disabled?

Reply:
It works now